src/app/app.component.html

```diff
@@ -1,2 +1,17 @@
-<router-outlet></router-outlet>
+<ul>
+  <li class="active"><a routerLink="/">Главная</a></li>
```

Comment:
Please use only one language in the project - Russian or English (better English)